src/test/java/seedu/tasklist/TaskListTest.java

```diff
@@ -0,0 +1,99 @@
+package seedu.tasklist;
+
+import org.junit.jupiter.api.Test;
+import seedu.command.AddCommand;
+import seedu.exception.ProjException;
+import seedu.tasks.Class;
+import seedu.tasks.TaskNonclass;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.common.Constants.TAB;
+
+public class TaskListTest {
+
+    private void addTestTask(TaskList tasks) {
+        tasks.addTask(new TaskNonclass("testTitle", "", "2020-06-10", "", "NUS", "", "Test"));
+        tasks.addTask(new Class("testClass", "", "1 3", "", "NTU NUS", "", "CLASS"));
+    }
+
+    @Test
+    public void addAndDeleteTask() {
+        TaskList tasks = new TaskList();
+        addTestTask(tasks);
+        String expectedTask = "[TEST] Title: testTitle | 2020-06-10 ( NUS )";
+        String expectedClass = "[CLASS] Title: testClass | MONDAY ( NTU ) | WEDNESDAY ( NUS )";
+        assertEquals(expectedTask,tasks.getTask(0).toString());
+        assertEquals(expectedClass,tasks.getTask(1).toString());
+
+        Integer output = tasks.getListSize();
+        Integer expected = 2;
+        assertEquals(output,expected);
+        tasks.deleteTask(1);
+        Integer outputAfterDelete = tasks.getListSize();
+        Integer expectedAfterDelete = 1;
+        assertEquals(outputAfterDelete,expectedAfterDelete);
+    }
+
+    @Test

```

Comment:
Error should be here, because this @Test was not removed @JTWang2000

Reply:
I have removed it via Git edit, checking to see if that's the cause now

Reply:
Problem resolved

Reply:
Thanks for your help! I really appreciate it.